modules/backup-restore/pages/enterprise-backup-restore.adoc

```diff
@@ -49,24 +46,24 @@ When you use this flag, `cbbackupmgr` backs up just the following:
 * GSI indexes
 * query UDFs 
 * scopes and collections definitions (Couchbase Server version 7.6 and later)
-* users (Couchbase Server version 7.6 and later)
-* user groups (Couchbase Server version 7.6 and later)
 * views
 
```

Comment:
Could a note be added here that cbbackupmgr does not back up Query function libraries, for example, javascript libraries created by users.

Reply:
I've added the note in. Are there any other notable things that cbbackupmgr doesn't back up? If so, we could have add another list.

Reply:
I think that the list of what's being backed up besides Data is fine.
The main confusion was with the description "Query UDFs" since that was ambiguous.
Thank you for the update.  I've approved the review.